resilience4j-cache/src/main/java/io/github/resilience4j/cache/internal/CacheImpl.java

```diff
@@ -93,7 +93,7 @@ private void putValueIntoCache(K cacheKey, V value) {
                 cache.put(cacheKey, value);
             }
         } catch (Exception exception){
-            LOG.warn(String.format("Failed to put a value into Cache %s", getName()), exception);
+            LOG.warn("Failed to put a value into Cache {}", getName(), exception);
```

Comment:
Hi,
sure that this works? Isn't the the exception treated as a param for the formatted string instead of anexception?

Reply:
Yes, as of slf4j 1.6.0, a parameter will be handled as an exception if it is the last argument. https://www.slf4j.org/faq.html#paramException

Reply:
Thx 👍